docs/how-to-use-and-FAQ/FAQ-ncnn-produce-wrong-result.md

```diff
@@ -1,6 +1,6 @@
 ### caffemodel should be row-major
 
-`caffe2ncnn` tool assumes the caffemodel is row-major (produced by c++ caffe train command).
+`caffe2ncnn` tool asumes the caffemodel is row-major (produced by c++ caffe train command).
```

Comment:
why is `assumes` a typo ?

Reply:
This was a mistake, my bad! I corrected it to be ready for the merge.
Have a nice day!